### PR TITLE
Hw-2

### PR DESCRIPTION
**Pull request by Kaijou88**

### CODE REVIEW

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 1]
```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return !(a ^ b ^ c ^ d) & (a | b | c | d) & ((a & b & c & d) == false);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BooleanExpression.java]: try to simplify

Comment (Kaijou88) [reply src/main/java/core/basesyntax/BooleanExpression.java]: done

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 2]
```diff
@@ -14,6 +14,8 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        // return (a?1:0)+(b?1:0)+(c?1:0)+(d?1:0) == 2;
+        //return !(a ^ b ^ c ^ d) & (a | b | c | d) & !(a & b & c & d);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BooleanExpression.java]: don't leave commented code

Comment (Kaijou88) [reply src/main/java/core/basesyntax/BooleanExpression.java]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**